pitopcli/oled.py

```diff
@@ -19,33 +19,62 @@ def run(self) -> int:
-                            default=False,
-                            )
-        parser.add_argument("--timeout", "-t",
-                            type=int,
-                            help="Set the timeout in seconds",
-                            default=10,
-                            )
-        parser.add_argument("--font-size", "-s",
-                            type=int,
-                            help="Set the font size",
-                            default=20,
-                            )
-        parser.add_argument("text",
-                            help="Set the text to write to screen",
-                            )
+        subparser = parser.add_subparsers(title="OLED screen utilities",
+                                          description="Set of utilities to use pi-top [4]'s OLED screen",
+                                          dest="oled_subcommand")
+
+        parser_draw = subparser.add_parser("draw", help="Draw text and images into the OLED")
+        parser_draw.add_argument("--force", "-f",
+                                 type=int,
+                                 help="Force the hub to give control of the OLED to the Pi",
+                                 default=False,
+                                 )
+        parser_draw.add_argument("--timeout", "-t",
+                                 type=int,
+                                 help="Set the timeout in seconds",
+                                 default=10,
+                                 )
+        parser_draw.add_argument("--font-size", "-s",
+                                 type=int,
+                                 help="Set the font size",
+                                 default=20,
+                                 )
+        parser_draw.add_argument("text",
+                                 help="Set the text to write to screen",
+                                 )
+
+        parser_capture = subparser.add_parser("capture", help="Capture images or videos of the OLED content")
+        parser_capture.add_argument("--force", "-f",
+                                    type=int,
+                                    help="Force the hub to give control of the OLED to the Pi",
+                                    default=False,
+                                    )
+        parser_capture.add_argument("--timeout", "-t",
+                                    type=int,
+                                    help="Set the timeout in seconds",
+                                    default=10,
+                                    )
+        parser_capture.add_argument("--font-size", "-s",
+                                    type=int,
+                                    help="Set the font size",
+                                    default=20,
+                                    )
+        parser_capture.add_argument("text",
+                                    help="Set the text to write to screen",
+                                    )
```

Comment:
I would empty the unused arguments for this section - perhaps we can implement the snapshot function quite easily? The canvas already has a "save to file" function!